### PR TITLE
Fix Requests on 0.6

### PR DESCRIPTION
**Pull request by Keno**

### CODE REVIEW

File: src/multipart.jl [hunk 1]
```diff
@@ -225,7 +225,7 @@ function prepare_multipart_request!(request, files)
         chunked = true
     end
 
-    datasizes = Array(Int,length(files))
+    datasizes = Array{Int}(length(files))
```
Comment (quinnj) [inline src/multipart.jl]: Isn't this supposed to be `Vector{Int}` now?

Comment (Keno) [inline src/multipart.jl]: either way is fine

**Comment by malmaud** [comment]
Can you explain/leave a comment on what was wrong?

**Comment by Keno** [comment]
The code assumed that the size of the underlying buffer was the same as the amount of data in the buffer, but as of recently we pre-allocate 32bytes.

**Pull request merged**